src_py/threads/__init__.py

```diff
@@ -106,15 +102,15 @@ def doit(x):
 
         wq = WorkerQueue(num_workers)
         t1 = time.time()
-        for xx in range(20):
-            print("active count:%s" % threading.activeCount())
+        for _ in range(20):
+            print(f"active count:{threading.activeCount()}")
```

Comment:
unrelated to this PR but... `threading.activeCount` is a deprecated function and `threading.active_count` is the new name for it

Reply:
I think we could probably do a separate PR pass on cleaning up places where pygame itself uses functions it has deprecated (there are a bunch in the tests if you run the test files one by one)